src/OrleansAWSUtils/Options/DynamoDBMembershipTableOptions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrleansAWSUtils.Configuration
+{
+    public class DynamoDBMembershipTableOptions
+    {
+        /// <summary>
+        /// Deployment Id.
+        /// </summary>
+        public string DeploymentId { get; set; }
```

Comment:
DeploymentId is a Global option, I'd not copy those around into different options, but give accept it from DI. For now it must be GlobalConfiguration, but having GlobalConfiguration as a dependency is better then replicating the same property into different Options classes.

What about extracting DeploymentId into some foreseeable class, like DeploymentOptions and pass it in as a dependency? Then later we can couple the other deployment related config properties there.

I guess based on @jdom's comment about "Table" we'll loose Table everywhere, so it will become DynamoDBMembershipOptions?

Reply:
I think this configuration should be in the Options for this provider, nevertheless I would add a default IPostConfigure to read it from the place were is globally defined, if it wasn't defined by the end user.
BTW, in this new configuration, we should drop the notion of DeploymentId. Probably call it ClusterId. What do you think @sergeybykov ?

Reply:
why this provider is different than other provide, that we don't pass deploymentId from cluster config?

Reply:
@jdom, just to confirm, you think DeploymentId should be in the Option for all MembershipTables? What's the reason of that? It seems to me DeploymentId is not a membershipTable specific concept.

Reply:
oh, sorry for the confusion, I think what @attilah suggested is correct too, and probably more appropriate than the PostConfigure suggestion, as I can't think of a scenario where you would want to configure a different ClusterId for the membership provider than the actual silo's ClusterId.
In this PR I'd start with injecting GlobalConfiguration into each provider, but eventually we can get it from a different typed option in a separate PR, once we extract that bit of configuration into a ClusterId property (maybe in the existing SiloIdentityOptions or some other)